grunt/config/copy.js

```diff
@@ -1,240 +1,207 @@
+        dest: '<%= outputdir %>fonts/',
+        filter: 'isFile',
+        rename: _.partial(collate, "fonts")
+      }]
+    },
+    menuFonts: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>menu/<%= menu %>/fonts/**'],
+        dest: '<%= outputdir %>fonts/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        menuAssets: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>menu/<%= menu %>/assets/**'],
-                    dest: '<%= outputdir %>assets/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "assets")
-                }
-            ]
+        rename: _.partial(collate, "fonts")
+      }]
+    },
+    themeAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>theme/<%= theme %>/assets/**'],
+        dest: '<%= outputdir %>assets/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        coreFonts: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>core/fonts/**'],
-                    dest: '<%= outputdir %>fonts/',
-                    filter: 'isFile',
-                    rename: _.partial(collate, "fonts")
-                }
-            ]
+        rename: _.partial(collate, "assets")
+      }]
+    },
+    themeFonts: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>theme/<%= theme %>/fonts/**'],
+        dest: '<%= outputdir %>fonts/',
+        filter: function(filepath) {
+          return grunt.config('helpers').includedFilter(filepath);
         },
-        menuFonts: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= sourcedir %>menu/<%= menu %>/fonts/**'],
-                    dest: '<%= outputdir %>fonts/',
-                    filter: function(filepath) {
-                        return grunt.config('helpers').includedFilter(filepath);
-                    },
-                    rename: _.partial(collate, "fonts")
-                }
-            ]
+        rename: _.partial(collate, "fonts")
+      }]
+    },
+    scriptLoader: {
+      files: [{
+        expand: true,
+        src: ['<%= sourcedir %>core/js/scriptLoader.js'],
+        dest: '<%= outputdir %>adapt/js/',
+        filter: 'isFile',
+        flatten: true
+      }]
+    },
+    libraries: {
+      files: [{
+        expand: true,
+        src: [
+          '<%= sourcedir %>core/js/libraries/**/*'
+        ],
+        dest: '<%= outputdir %>libraries/',
+        rename: _.partial(collate, "libraries")
+      }]
+    },
+    required: {
+      files: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?